### ISSUE TITLE
[BUG] Backup Album selection freezes on the Panorama smart album

### ISSUE THREAD

**Issue by partnerinflight** [immich-app/immich#6051]
### The bug

1. Install Immich mobile app on an iOS 17.2 device with an iCloud Photo Sync enabled, and the Panorama album containing photos
2. Go to Backup -> Album Selection. 
Result: The spin indicator continues indefinitely. List of albums never shows up
Expected Result: List of albums on device should show up.

### The OS that Immich Server is running on

TrueNAS SCALE (Bluefin or latest)

### Version of Immich Server

1.91.4

### Version of Immich Mobile App

1.91.0.131

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
using the Immich SCALE app
```

### Your .env content

```Shell
Using the Immich SCALE app
```

### Reproduction steps

```bash
1. Install Immich
2. Go to Album Selection under Backup

Result: Albums never show up
```

### Additional information

I've actually built and debugged the app locally. There's an error that happens in photo-manager when immich tries to get the Thumbnail for the Panoramas album. I was able to get around the issue by simply skipping the thumbnail retrieval for that album. Might consider doing something like this:

backup.provider.dart:240:

 // The Panoramas album gets stuck trying to generate
        // thumbnail, and the thumbnailDataWithSize call never returns
        // this causes a permanent loop in the UI.
        if (album.darwinSubtype == PMDarwinAssetCollectionSubtype.smartAlbumPanoramas) {
          availableAlbum = availableAlbum.copyWith();
        } else {
          try {
            final thumbnailData = await thumbnailAsset
                .thumbnailDataWithSize(const ThumbnailSize(512, 512));
            availableAlbum =
                availableAlbum.copyWith(thumbnailData: thumbnailData);
          } catch (e, stack) {
            log.severe(
              "Failed to get thumbnail for album ${album.name}",
              e.toString(),
              stack,
            );
          }
        }

**Comment by pbedmar** [comment]
This only happens to me when I give Immich access to the complete photos library. If I only give permission to some concrete photos, this problem disappears.

**Comment by partnerinflight** [comment]
Yeah, but doesn't that sort of defeat the purpose? I'd want Immich to be backing up the full library, not a subset...

**Comment by alextran1502** [comment]
This is one interesting data point for us to solve this issue. Do you see any error logs if you allow to select the panorama album only?

**Comment by pbedmar** [comment]
I can't check logs because I'm running an old version of Xcode. But if I select the panorama album, when I go back to the timeline and return to the tab to select albums, it gets automatically unselected.

**Comment by alextran1502** [comment]
There is a logs in the Immich mobile app, you can access it by tapping the profile picture

**Comment by pbedmar** [comment]
The test I described in my previous message was done with no connection to the server. A "Selected album not found" error was shown in the log. 

If I try the same with server connection, no errors are shown.

**Comment by dereitz** [comment]
I just installed Immich and believe I'm running into this same issue.  I granted full access and hoped to synchronize all my photos to Immich.  How can I help with troubleshooting?  I currently have 9,177 photos and 345 videos in my library. I checked the log in the iOS app and I don't see anything obvious.  It does say it found 366 local albums (no idea why there would be so many?), and of course the says "no assets are selected for back up" since I haven't been able to select anything for backup yet.  Anyway, please let me know if/how I can help troubleshoot.  Thanks!

**Comment by marcelo-mbwr** [comment]
Any news on this topic? Half of my family has this problem. All IOS version 17.2.1.

**Comment by jrasm91** [comment]
Is this still an issue?

**Comment by pbedmar** [comment]
Not for me anymore. You can probably close it.
